src/app/components/search/CustomCourseSelection.tsx

```diff
@@ -48,7 +48,7 @@ export default function CustomCourseSelection() {
   };
 
   return (
-    <Card className="w-80 mb-2 mr-2" variant="plain">
+    <Card className="w-80 mt-2 mr-2" variant="plain">
```

Comment:
Make this card have a minimum height so the left side of the UI isn't shifting when I'm adding and deleting custom courses